### DISCUSSION TITLE
Validation error

### DISCUSSION THREAD

**Discussion by elchanly** [zachowj/node-red-contrib-home-assistant-websocket#1255] [Q&A]
im having this strange error after upgrading to 62.3:
it happens in quite a few nodes (500 more or less), even if i try to replace them with new ones from the left palette
I had problems when upgrading to 62.3: i upgraded and config node seemed to dissapear. I read docs and found that node red wasnt updated to 3.1.1, upgraded and seemed to work until this flood of warnings.
It works, i mean everything seems to do what is supposed to, buttt there are 500+ nodes with this. What can I do????
Thanks!!!

![image](https://github.com/zachowj/node-red-contrib-home-assistant-websocket/assets/49251727/64af1b23-1a61-4f33-bf82-d27b75af7182)

**Comment by zachowj** [comment]
Is it only call-service nodes?

**Comment by elchanly** [comment]
It was in almost every ha node.
It seems to be node-red related, as it dissapeared this morning. I always restart nodered overnight, maybe that solved the issue. Its kind of strange, i restarted nr manually several times... 
idk, stranger things.
Thanks anyway <3
